test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/project.json

```diff
@@ -1,22 +1,18 @@
 {
   "dependencies": {
     "xunit": "2.1.0",
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "NETStandard.Library": "1.5.0-*"
   },
   "frameworks": {
-    "netcoreapp1.0": {
```

Comment:
After I changed it back to netcoreapp1.0 and fixed KoreBuild I now see this: 
Deps file [C:\Source\IISIntegration\test\dotnet-publish-iis.Tests\bin\Debug\netcoreapp1.0\win7-x64\dotnet-publish-iis.Tests.deps.json] specified but doesn't exist

@pakrym  will look into it but I wonder how you got past that?